documentation/Environments.md

```diff
@@ -0,0 +1,66 @@
+# Authentication service environments
+
+The auth service can support multiple environments, allowing it to service multiple hosts.
```

Comment:
The phrase "multiple hosts" can be a bit ambiguous.  Maybe something like multiple UI domains or multiple UI deployments.

Reply:
How about

```
The auth service can support multiple environments, allowing it to service multiple domains -
for a KBase example, `narrative.kbase.us`, `appdev.kbase.us`, and `narrative-dev.kbase.us`
are all served by the same auth service. This allows sharing user accounts and tokens between
environments and deploying alternate versions of UIs running on the same backend.
```

Reply:
Updated